controllers/operands/kubevirt.go

```diff
@@ -232,8 +240,8 @@ func (*kubevirtHooks) updateCr(req *common.HcoRequest, Client client.Client, exi
 
 func (*kubevirtHooks) justBeforeComplete(_ *common.HcoRequest) { /* no implementation */ }
 
-func NewKubeVirt(hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtcorev1.KubeVirt, error) {
-	config, err := getKVConfig(hc)
+func NewKubeVirt(ctx context.Context, client client.Client, hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtcorev1.KubeVirt, error) {
+	config, err := getKVConfig(ctx, client, hc, getNamespace(hc.Namespace, opts))
```

Comment:
We don't need the `getNamespace` here. Please drop.

Reply:
Done.